forgevtt-module.js

```diff
@@ -2326,7 +2326,19 @@ class ForgeVTT_FilePicker extends FilePicker {
         if (target == null) {
             target = "";
             if (body.uuid) {
-                target = `worlds/${game.world.id || game.world.name}`
+                target = `worlds/${game.world.id || game.world.name}`;
+            }
+            if (event && event.type === "paste") {
```

Comment:
Where is this `event` variable coming from? Sounds like undefined... 
I think maybe it's one of the optional arguments but you didn't add it to the function parameters?

Reply:
> Where is this `event` variable coming from? Sounds like undefined... I think maybe it's one of the optional arguments but you didn't add it to the function parameters?

It gets added into the scope by prosemirror, and remains defined in that scope even though it is not explicitly passed as a function parameter. 

![image](https://github.com/ForgeVTT/fvtt-module-forge-vtt/assets/34374883/c41e67d1-d290-4af5-b64d-8c435266bef5)

If we opt for always unique paste/drag events (as Foundry does) then we can do this based off of the `body.uuid` (journal and journal page IDs, not image ID), as Foundry does, and avoid needing to use regex or check events

Reply:
huh... unless it's a global variable, I'm not sure I understand how it works. It's there though, so not a bug at least 🤷